src/Middleware/SpaServices.Extensions/src/AngularCli/AngularCliMiddleware.cs

```diff
@@ -63,13 +64,18 @@ internal static class AngularCliMiddleware
         }
 
         private static async Task<AngularCliServerInfo> StartAngularCliServerAsync(
-            string sourcePath, string scriptName, string pkgManagerCommand, ILogger logger)
+            string sourcePath, string scriptName, string pkgManagerCommand, int portNumber, ILogger logger)
         {
-            var portNumber = TcpPortFinder.FindAvailablePort();
+            string scriptArgs = null;
+            if (portNumber == 0)
```

Comment:
```suggestion
            if (portNumber == default(int))
```

That's just a nitpick for clarity though.

Reply:
✔️